src/components/card/Card.jsx

```diff
@@ -49,23 +49,71 @@ class Card extends Component {
     return (
       <div>
         {this.props.labels.map(label => (
-          <Tag color={label.color} key={this.props.id + label.id}>
+          <Tag color={label.color} key={this.props.id + label.id} className="label">
```

Comment:
For `className`, prepend the value with the component name. At the building time, all CSS files will be merge and minified in just one file. So you can't just take the `label` for you, maybe something else use it already. I suggest `cardLabel`.

You need to do that for all `className` used in this file.